TLM/TLM/Util/TrackUtils.cs

```diff
@@ -17,22 +17,31 @@ internal static class TrackUtils {
             VehicleInfo.VehicleType.Monorail;
 
         internal const NetInfo.LaneType ROAD_LANE_TYPES = LaneArrowManager.LANE_TYPES;
+        internal const NetInfo.LaneType ROUTED_ROAD_LANE_TYPES = LaneArrowManager.LANE_TYPES | NetInfo.LaneType.CargoVehicle;
```

Comment:
maybe we should just use the one in routing manger instead of duplicate values.

Reply:
Hmm, I was thinking about that, but on the other hand we already use `MatchesTrack` from here in the Routing Manager...
Anyways.. TrackUtils class is not the best class name if we collect both, track and road helpers 😄

Reply:
I was struggling with the naming of this class. It was to distinguish track from road.

Reply:
I'll leave it like it is for now, there are some plans for mixing trains on road, so maybe we will need to rethink the whole logic if needed. It's not a big issue.